content/en/docs/gke/pipelines/authentication-sdk.md

```diff
@@ -0,0 +1,102 @@
++++
+title = "Connecting to Kubeflow Pipelines on Google Cloud using the SDK"
+description = "How to connect to different Kubeflow Pipelines installations on Google Cloud with Kubeflow Pipelines SDK?"
+weight = 20
++++
+
+This guide describes how to connect to your Kubeflow Pipelines on Google CLoud using [the Kubeflow Pipelines SDK](/docs/pipelines/sdk/sdk-overview/).
+
+## Before you begin
+
+* You need a Kubeflow Pipelines deployment on Google Cloud using one of the [installation options](/docs/pipelines/installation/overview/).
+* You need to [install Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/).
+
+## How SDK connects to Kubeflow Pipelines API
+
+Kubeflow Pipelines has an API service named `ml-pipeline-ui` in the Kubernetes
```

Comment:
Suggested revision(it is ambiguous):

Kubeflow Pipelines has an API service named `ml-pipeline-ui` in the Kubernetes namespace you deployed it to.

->

Kubeflow Pipelines includes an API service named `ml-pipeline-ui`. The `ml-pipeline-ui` API service is deployed in the same Kubernetes namespace you deployed Kubeflow Pipielines in.